### ISSUE TITLE
Button API token not working

### ISSUE THREAD

**Issue by ferrigno** [oda-hub/frontend-astrooda#43]
I tried to click on "API token" at the instance 
https://frontend.staging.odahub.io/cdci/astrooda/
and got the following error
(if expected, please close the issue, when it is fixed)
<pre>
An AJAX HTTP error occurred.
HTTP Result Code: 404
Debugging information follows.
Path: modal_forms/ajax/webform/384
StatusText: Not Found
ResponseText: 
Page not found | Astro Online Data Analysis
@import url("https://frontend.staging.odahub.io/cdci/astrooda/modules/system/system.base.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/jquery_update/replace/ui/themes/base/minified/jquery.ui.core.min.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/jquery_update/replace/ui/themes/base/minified/jquery.ui.theme.min.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/jquery_update/replace/ui/themes/base/minified/jquery.ui.resizable.min.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/modules/overlay/overlay-parent.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/date/date_api/date.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/date/date_popup/themes/datepicker.1.7.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/modules/field/theme/field.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/modules/node/node.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/toc_node/css/toc_node.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/views/css/views.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/ckeditor/css/ckeditor.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/admin_menu/admin_menu.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/admin_menu/admin_menu_toolbar/admin_menu_toolbar.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/modules/shortcut/shortcut.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/ctools/css/ctools.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/ctools/css/modal.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/modal_forms/css/modal_forms_popup.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/devel/devel_krumo.css?quqi2z");
ul.krumo-node{margin:0px;padding:0px;}ul.krumo-node ul{margin-left:20px;}* html ul.krumo-node ul{margin-left:24px;}div.krumo-root{border:solid 1px black;margin:1em 0em;}ul.krumo-first{font:normal 12px arial;border:solid 2px white;border-top-width:1px;background:url(https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/devel/krumo/skins/default/bg.gif);}li.krumo-child{display:block;list-style:none;padding:0px;margin:0px;overflow:hidden;}div.krumo-element{cursor:default;line-height:24px;display:block;clear:both;white-space:nowrap;border-top:solid 1px white;background:#E8E8E8;padding-left:10px;}* html div.krumo-element{padding-bottom:3px;}a.krumo-name{color:#2C5858;font:bold 13px Arial;}a.krumo-name span.krumo-big{font:bold 20pt Georgia;line-height:14px;position:relative;top:2px;left:-2px;}* html a.krumo-name span.krumo-big{font:bold 19pt Georgia;top:5px;left:0px;line-height:9px;height:12px;padding:0px;margin:0px;}div.krumo-expand{background:#CCCCCC;cursor:pointer;}div.krumo-hover{background:#B7DBDB;}div.krumo-preview{font:normal 13px courier new;padding:5px 5px 14px 5px;background:white;border-top:0px;overflow:auto;white-space:pre;}* html div.krumo-preview{padding-top:2px;}li.krumo-footnote{background:white;padding:2px 5px;list-style:none;border-top:solid 1px #bebebe;margin-top:2px;cursor:default;}* html li.krumo-footnote{line-height:13px;}div.krumo-version{float:right;}li.krumo-footnote h6{font:bold 11px verdana;margin:0px;padding:0px;color:#366D6D;display:inline;}* html li.krumo-footnote h6{margin-right:3px;}li.krumo-footnote a{font:bold 10px arial;color:#434343;text-decoration:none;}li.krumo-footnote a:hover{color:black;}li.krumo-footnote span.krumo-call{font:normal 11px verdana;position:relative;top:1px;}li.krumo-footnote span.krumo-call code{font-weight:bold;}div.krumo-title{font:normal 11px verdana;position:relative;top:9px;cursor:default;line-height:2px;}strong.krumo-array-length,strong.krumo-string-length{font-weight:normal;}
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/modules/eu_cookie_compliance/css/eu_cookie_compliance.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/default/files/ctools/css/2649ce92228a3791844ace042f5c5e23.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/themes/bootstrap/css/3.3.5/overrides.min.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/themes/bootstrap_astrooda/css/bootstrap.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/themes/bootstrap_astrooda/css/skin.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/themes/bootstrap_astrooda/fontawesome/css/font-awesome.min.css?quqi2z");
@import url("https://frontend.staging.odahub.io/cdci/astrooda/sites/all/themes/bootstrap_astrooda/css/custom.css?quqi2z");
/**
* JavaScript routines for Krumo
*
* @link http://sourceforge.net/projects/krumo
*/
/////////////////////////////////////////////////////////////////////////////
/**
* Krumo JS Class
*/
function krumo() {
}
// -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
/**
* Add a CSS class to an HTML element
*
* @param HtmlElement el
* @param string className
* @return void
*/
krumo.reclass = function(el, className) {
if (el.className.indexOf(className)  -1) {
el.className = el.className.replace(className, '');
}
}
// -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
/**
* Toggle the nodes connected to an HTML element
*
* @param HtmlElement el
* @return void
*/
krumo.toggle = function(el) {
var ul = el.parentNode.getElementsByTagName('ul');
for (var i=0; i
jQuery.extend(Drupal.settings, {"basePath":"\/cdci\/astrooda\/","pathPrefix":"","ajaxPageState":{"theme":"bootstrap_astrooda","theme_token":"-0iewEE1-vs6wp-c6MzwkB9_dtbigynwaT9u_5gqAE8","jquery_version":"1.9","js":{"0":1,"sites\/all\/modules\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/all\/themes\/bootstrap\/js\/bootstrap.js":1,"sites\/all\/modules\/jquery_update\/replace\/jquery\/1.9\/jquery.min.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.effect.min.js":1,"sites\/all\/modules\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.effect-highlight.min.js":1,"sites\/all\/modules\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/jquery_update\/replace\/misc\/1.9\/jquery.ba-bbq.min.js":1,"sites\/all\/modules\/jquery_update\/replace\/misc\/1.9\/overlay-parent.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/jquery_update\/js\/jquery_update.js":1,"https:\/\/cdn.jsdelivr.net\/npm\/bootstrap@3.3.5\/dist\/js\/bootstrap.js":1,"sites\/all\/modules\/admin_menu\/admin_menu.js":1,"sites\/all\/modules\/admin_menu\/admin_menu_toolbar\/admin_menu_toolbar.js":1,"sites\/all\/modules\/modal_forms\/js\/modal_forms_contact.js":1,"sites\/all\/themes\/bootstrap\/js\/misc\/_progress.js":1,"sites\/all\/modules\/ctools\/js\/modal.js":1,"sites\/all\/modules\/modal_forms\/js\/modal_forms_popup.js":1,"sites\/all\/modules\/devel\/devel_krumo.js":1,"1":1,"sites\/all\/themes\/bootstrap\/js\/misc\/ajax.js":1,"sites\/all\/themes\/bootstrap\/js\/modules\/ctools\/js\/modal.js":1},"css":{"modules\/system\/system.base.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"modules\/overlay\/overlay-parent.css":1,"sites\/all\/modules\/date\/date_api\/date.css":1,"sites\/all\/modules\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/toc_node\/css\/toc_node.css":1,"sites\/all\/modules\/views\/css\/views.css":1,"sites\/all\/modules\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/admin_menu\/admin_menu.css":1,"sites\/all\/modules\/admin_menu\/admin_menu_toolbar\/admin_menu_toolbar.css":1,"modules\/shortcut\/shortcut.css":1,"sites\/all\/modules\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/ctools\/css\/modal.css":1,"sites\/all\/modules\/modal_forms\/css\/modal_forms_popup.css":1,"sites\/all\/modules\/devel\/devel_krumo.css":1,"0":1,"sites\/all\/modules\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"public:\/\/ctools\/css\/2649ce92228a3791844ace042f5c5e23.css":1,"https:\/\/cdn.jsdelivr.net\/npm\/bootstrap@3.3.5\/dist\/css\/bootstrap.css":1,"sites\/all\/themes\/bootstrap\/css\/3.3.5\/overrides.min.css":1,"sites\/all\/themes\/bootstrap_astrooda\/css\/bootstrap.css":1,"sites\/all\/themes\/bootstrap_astrooda\/css\/skin.css":1,"sites\/all\/themes\/bootstrap_astrooda\/fontawesome\/css\/font-awesome.min.css":1,"sites\/all\/themes\/bootstrap_astrooda\/css\/custom.css":1}},"user_uid":"6","CToolsModal":{"loadingText":"Loading...","closeText":"Close Window","closeImage":"\u003Cimg typeof=\u0022foaf:Image\u0022 class=\u0022img-responsive\u0022 src=\u0022https:\/\/frontend.s
</pre>

**Comment by volodymyrss** [comment]
I suspect something needs to be provided in database, even with the necessary module installed, this is lacking.

**Comment by motame** [comment]
It must be a dependency issue.
A self-contained system : 
- frontend-drupal7-for-astrooda
- frontend-db-drupal7-for-astrooda
- frontend-astrooda
- frontend-bootstrap_astrooda
